documentation/book/proc-deploying-kafka-connect-kubernetes.adoc

```diff
@@ -3,20 +3,24 @@
 // assembly-kafka-connect.adoc
 
 [id='deploying-kafka-connect-kubernetes-{context}']
-= Deploying Kafka Connect to {KubernetesName}
+= Deploying Kafka Connect to your {KubernetesName} cluster
+
+To allow you to build connectors for your Kafka cluster, you can deploy a Kafka Connect cluster to your {KubernetesName} cluster by using the Cluster Operator.
 
 .Prerequisites
 
-* Before deploying Kafka Connect, the Cluster Operator must be deployed.
+* xref:deploying-cluster-operator-kubernetes-{context}[Deploying the Cluster Operator to {KubernetesName}]
```

Comment:
I believe you want to replace {context} here with whatever the context was on the page you're linking to